project-base/assets/js/frontend/components/lazyLoadInit.js

```diff
@@ -8,10 +8,10 @@ new MiniLazyload({
     placeholder: '/assets/frontend/images/noimage.png'
 }, '', MiniLazyload.IGNORE_NATIVE_LAZYLOAD);
 
-export function lazyLoadCall (container) {
```

Comment:
The old usage is still present in `upgrade/upgrade-instruction-for-webpack.md` and `doc/frontend/lazyload.md`

Reply:
3e23cf9e30906af7b6772be85608e1ec14c0ea24